src/sql/db_connection_pool/dbconnection/mysqlconn.rs

```diff
@@ -43,6 +44,26 @@ pub struct MySQLConnection {
     pub conn: Arc<Mutex<Conn>>,
 }
 
+impl MySQLConnection {
+    fn column_meta_query_for(tbl: &TableReference) -> String {
+        let q = MySqlDialect {}
+            .identifier_quote_style("") // parameter unimportant for `MySqlDialect`.
```

Comment:
Why we override MySQL default `identifier_quote_style`?

```
impl Dialect for MySqlDialect {
    fn identifier_quote_style(&self, _: &str) -> Option<char> {
        Some('`')
    }
```

Reply:
Might be better to just add utility function with own implementation w/o using MySqlDialect?